tests/testthat/test-dataframe.R

```diff
@@ -157,10 +157,7 @@ test_that("DataFrame, custom schema", {
   )
   # incorrect datatype
   expect_grepl_error(pl$DataFrame(x = 1, schema = list(schema = foo)))
-  expect_grepl_error(
-    pl$DataFrame(x = 1, schema = list(x = "foo")),
-    "expected RPolarsDataType"
-  )
+  expect_grepl_error(pl$DataFrame(x = 1, schema = list(x = "foo")))
```

Comment:
This change (and others below) make the test pass but they degrade the error messages. For example, instead of 
`expected RPolarsDataType` we now have `Expected ExternalPtr got Strings` which is quite cryptic. Is there a way to not modify those error messages?